sys/net/gnrc/link_layer/lwmac/include/lwmac_internal.h

```diff
@@ -0,0 +1,408 @@
+{
+    return (dev->mac_info & GNRC_NETDEV_LWMAC_QUIT_RX);
+}
+
+/**
+ * @brief set the duty-cycle-active flag of Lwmac
+ *
+ * @param[in] dev          ptr to netdev device
+ * @param[in] active       value for Lwmac duty-cycle-active flag
+ *
+ */
+static inline void gnrc_netdev_lwmac_set_dutycycle_active(gnrc_netdev_t *dev, bool active)
+{
+    if (active) {
+        dev->lwmac.lwmac_info |= LWMAC_DUTYCYCLE_ACTIVE;
+    }
+    else {
+        dev->lwmac.lwmac_info &= ~LWMAC_DUTYCYCLE_ACTIVE;
+    }
+}
+
+/**
+ * @brief get the duty-cycle-active flag of Lwmac
+ *
+ * @param[in] dev          ptr to netdev device
+ *
+ * @return                 true if active
+ * @return                 false if not active
+ */
+static inline bool gnrc_netdev_lwmac_get_dutycycle_active(gnrc_netdev_t *dev)
+{
+    return (dev->lwmac.lwmac_info & LWMAC_DUTYCYCLE_ACTIVE);
+}
+
+/**
+ * @brief set the needs-rescheduling flag of Lwmac
+ *
+ * @param[in] dev          ptr to netdev device
+ * @param[in] reschedule   value for Lwmac needs-rescheduling flag
+ *
+ */
+static inline void gnrc_netdev_lwmac_set_reschedule(gnrc_netdev_t *dev, bool reschedule)
+{
+    if (reschedule) {
+        dev->lwmac.lwmac_info |= LWMAC_NEEDS_RESCHEDULE;
+    }
+    else {
+        dev->lwmac.lwmac_info &= ~LWMAC_NEEDS_RESCHEDULE;
+    }
+}
+
+/**
+ * @brief get the needs-rescheduling flag of Lwmac
+ *
+ * @param[in] dev          ptr to netdev device
+ *
+ * @return                 true if needs rescheduling
+ * @return                 false if no need for rescheduling
+ */
+static inline bool gnrc_netdev_lwmac_get_reschedule(gnrc_netdev_t *dev)
+{
+    return (dev->lwmac.lwmac_info & LWMAC_NEEDS_RESCHEDULE);
+}
+
+/**
+ * @brief Parse an incoming packet and extract important information.
+ *
+ *        Copies addresses into @p info, but header points inside @p pkt.
+ *
+ * @param[in]   pkt             packet that will be parsed
+ * @param[out]  info            structure that will hold parsed information
+ *
+ * @return                      0 if correctly parsed
+ * @return                      <0 on error
+ */
+int _parse_packet(gnrc_pktsnip_t *pkt, lwmac_packet_info_t *info);
+
+/**
+ * @brief Shortcut to get the state of netdev.
+ *
+ * @param[in]   gnrc_netdev    gnrc_netdev structure
+ *
+ * @return                     state of netdev
+ */
+netopt_state_t _get_netdev_state(gnrc_netdev_t *gnrc_netdev);
+
+/**
+ * @brief Shortcut to set the state of netdev
+ *
+ * @param[in]   gnrc_netdev    gnrc_netdev structure
+ * @param[in]   devstate       new state for netdev
+ */
+void _set_netdev_state(gnrc_netdev_t *gnrc_netdev, netopt_state_t devstate);
+
+/**
+ * @brief Convert RTT ticks to device phase
+ *
+ * @param[in]   ticks    RTT ticks
+ *
+ * @return               device phase
+ */
+static inline uint32_t _ticks_to_phase(uint32_t ticks)
+{
+    assert(LWMAC_WAKEUP_INTERVAL_US != 0);
+
+    return (ticks % RTT_US_TO_TICKS(LWMAC_WAKEUP_INTERVAL_US));
```

Comment:
I guess this is due to that there is no RTT definitions on iotlab-m3 ? Since, if there is no definition of RTT, then, `RTT_US_TO_TICKS()` should always return 0. i.e., `RTT_FREQUENCY` in the following phase is 0.
```
#define RTT_US_TO_TICKS(us)     ((uint32_t)((uint64_t)(us) * RTT_FREQUENCY / 1000000UL))
```

I currently have no idea how to deal with that, some tips? Exclude iotlab-m3 for checking?